src/handlers/ens.ts

```diff
@@ -81,3 +89,143 @@ export const fetchRegistrationDate = async (recipient: any) => {
     return registrationDate;
   }
 };
+
+export const estimateENSRegisterWithConfigGasLimit = async (
+  name: string,
+  ownerAddress: string,
+  duration: number,
+  rentPrice: string
+) =>
+  estimateENSTransactionGaslimit({
+    duration,
+    name,
+    ownerAddress,
+    rentPrice,
+    type: ENSRegistrationTransactionType.REGISTER_WITH_CONFIG,
+  });
+
+export const estimateENSCommitGasLimit = async (
+  name: string,
+  ownerAddress: string,
+  duration: number,
+  rentPrice: string
+) =>
+  estimateENSTransactionGaslimit({
+    duration,
+    name,
+    ownerAddress,
+    rentPrice,
+    type: ENSRegistrationTransactionType.COMMIT,
+  });
+
+export const estimateENSSetTextGasLimit = async (
+  name: string,
+  recordKey: string,
+  recordValue: string
+) =>
+  estimateENSTransactionGaslimit({
+    name,
+    records: {
+      coinAddress: null,
+      contentHash: null,
+      ensAssociatedAddress: null,
+      text: [
+        {
+          key: recordKey,
+          value: recordValue,
+        },
+      ],
+    },
+    type: ENSRegistrationTransactionType.SET_TEXT,
+  });
+
+export const estimateENSSetNameGasLimit = async (
+  name: string,
+  ownerAddress: string
+) =>
+  estimateENSTransactionGaslimit({
+    name,
+    ownerAddress,
+    type: ENSRegistrationTransactionType.SET_NAME,
+  });
+
+export const estimateENSMulticallGasLimit = async (
+  name: string,
+  records: ENSRegistrationRecords
+) =>
+  estimateENSTransactionGaslimit({
+    name,
+    records,
+    type: ENSRegistrationTransactionType.MULTICALL,
+  });
+
+export const estimateENSTransactionGaslimit = async ({
```

Comment:
nitpick but maybe `estimateENSTransactionGaslimit` should be `estimateENSTransactionGasLimit` (capital L) so it matches the capitalization of the other functions

Reply:
👌